engineapi/engineapi/contracts_actions.py

```diff
@@ -507,6 +517,21 @@ def list_call_requests(
             CallRequest.expires_at > func.now(),
         )
 
+    # If user id not specified, do not show call_requests before live_at.
+    # Otherwise check show_before_live_at argument from query parameter
+    if metatx_requester_id is not None:
+        query = query.filter(
+            CallRequest.metatx_requester_id == metatx_requester_id,
+        )
+        if not show_before_live_at:
```

Comment:
Do it actualy must be out of that if?
Current logic kind of unintuitive if  set **show_before_live_at:** to true without `metatx_requester_id` it still will apply that filter on else.